internal/deviations/deviations.go

```diff
@@ -745,3 +745,8 @@ func Ipv6RouterAdvertisementConfigUnsupported(dut *ondatra.DUTDevice) bool {
 func PrefixLimitExceededTelemetryUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetPrefixLimitExceededTelemetryUnsupported()
 }
+
+// SshServerCountersUnsupported is to skip checking ssh server counters.
+func SshServerCountersUnsupported(dut *ondatra.DUTDevice) bool {
```

Comment:
the static-analysis thing here seems reasonable.

Reply:
copy thats an easy one :D will sort it!